dev/tests/proxy.rs

```diff
@@ -19,6 +19,18 @@ use std::net::SocketAddrV4;
 use std::{net::SocketAddr, time::Duration};
 use tokio::time::Instant;
 
+// websocket related
```

Comment:
Rather than segmenting out imports, we just use `rustfmt`'s organization. To achieve this, remove lines 21 and 22 so that there's no gap in the imports, and run `cargo fmt` in the root or `dev/` directory.